pkg/k8s/watchers/watcher.go

```diff
@@ -342,6 +342,9 @@ func (k *K8sWatcher) InitK8sSubsystem(ctx context.Context) <-chan struct{} {
 			// We need to know about active local redirect policy services
 			// before BPF LB datapath is synced.
 			k8sAPIGroupCiliumLocalRedirectPolicyV2,
+			// We need to know the node labels to populate the host endpoint
+			// labels.
+			k8sAPIGroupNodeV1Core,
```

Comment:
@aanm Won't this cause scalability issues? IIRC you moved this away to cilium-operator, as  cilium-agent was not able to keep up with Node events.

Reply:
In case it matters: we're only watching for Node events for the local node.

Reply:
Yeah it's like Paul said, it's ok because we are only watching for 1 node.